README.md

```diff
@@ -37,7 +37,11 @@ Russian translation is maintained by Translation Gang.
 
 ### Korean
 
-TODO: provide information here
+Korean translation is maintained by Vue.js Korea.
+한국어 번역은 Vue.js 한국어 사용자 모임에서 관리합니다.
```

Comment:
I think that you don't have to mention with korean language, because `vuejs/vuejs.org` is main site with english.